packages/cloud2edge/values.yaml

```diff
@@ -23,9 +23,18 @@ honoConnection:
 # Configuration properties for Eclipse Hono.
 hono:
 
+  kafka:
+    auth:
+      sasl:
+        jaas:
+          clientUsers:
+            - "ditto-c2e"
+          clientPasswords:
+            - "verysecret"
```

Comment:
I am not sure if this simply *overwrites* the credentials defined in the Hono chart of if it *adds* the credentials to the ones defined in the Hono chart. In any case, we need to make sure that the credentials from the Hono chart are being kept as well ...

Reply:
I will check in detail how helm behaves in this case.
It seems though on a first look of the kafka logs that hono is accessing it:
`INFO Created log for partition hono.command_internal.HonoAMQPAdapter-e7104df7-34b9-408c-b158-ba7f89fb44cd-0 in /bitnami/kafka/data/hono.command_internal.HonoAMQPAdapter-e7104df7-34b9-408c-b158-ba7f89fb44cd-0 with properties {} (kafka.log.LogManager)`
But like I said - will look a bit deeper.

Reply:
checked and indeed it overwrites the values from hono.
So either I put both in values of cloud2edge or find another way to do this.

Reply:
> ... it works because hono takes the first values

Yes, then I guess we are all good. As long as these credentials match the ones set in `honoConnection` we should be fine ...

Reply:
FMPOV we should keep both sets of credentials so that the Hono command line client can still successfully connect to the Kafka broker. The `ditto-c2e/verysecret` credentials should be defined second in order not to break things ...

Reply:
ok, I've added them like requested also tested a deployment and confirmed that hono-secret is indeed the first one in the secret.